tests/TestCase.php

```diff
@@ -2,44 +2,23 @@
 
 namespace Wulfheart\PrettyRoutes\Tests;
 
-use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Routing\Controller;
 use Orchestra\Testbench\TestCase as Orchestra;
 use Wulfheart\PrettyRoutes\PrettyRoutesServiceProvider;
 
-class TestCase extends Orchestra
+abstract class TestCase extends Orchestra
 {
-    public function setUp(): void
+    protected function getPackageProviders($app): array
     {
-        parent::setUp();
-
-        Factory::guessFactoryNamesUsing(
-            fn (string $modelName) => 'Spatie\\PrettyRoutes\\Database\\Factories\\'.class_basename($modelName).'Factory'
-        );
-    }
-
-    protected function getPackageProviders($app)
-    {
-        return [
-            PrettyRoutesServiceProvider::class,
-        ];
+        return [PrettyRoutesServiceProvider::class];
     }
 
-    public function getEnvironmentSetUp($app)
+    protected function getEnvironmentSetUp($app): void
     {
-        $app['config']->set('database.default', 'sqlite');
-        $app['config']->set('database.connections.sqlite', [
-            'driver' => 'sqlite',
-            'database' => ':memory:',
-            'prefix' => '',
-        ]);
-
-        /*
-        include_once __DIR__.'/../database/migrations/create_pretty_routes_table.php.stub';
-        (new \CreatePackageTable())->up();
-        */
+        putenv('COLUMNS=120');
```

Comment:
Here we define a fixed terminal width now for all tests.

Reply:
Nice!